packages/slider/src/HandleController.ts

```diff
@@ -208,7 +208,7 @@ export class HandleController implements Controller {
         handles.forEach((handle, index) => {
             /* c8 ignore next */
             if (!handle.handleName?.length) {
-                handle.name = `handle${index + 1}`;
+                handle.handleName = `handle${index + 1}`;
```

Comment:
This seems weird in that `handle.handleName` is _only_ a getter for `handle.name`...

Reply:
https://github.com/adobe/spectrum-web-components/blob/d23f735db3cea01c2243e0485448d1598c6a8692/packages/slider/src/SliderHandle.ts#L80-L82

Reply:
Should've said, my "fix" is wrong (in editing it while I was waiting for CLA approval I removed that minor detail) but I'm not sure why the handle's `name` property isn't being initialized properly.

Reply:
Could you share into the issue more info around how you add the DOM to the page, etc.?

As this change isn't right, let's close this PR and discuss finding the source of this in the issue.